Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -140,65 +145,52 @@ open class _QuotedChatMessageView<ExtraData: ExtraDataTypes>: _View, ThemeProvid
             : appearance.colorPalette.highlightedAccentBackground1
 
         setAvatar(imageUrl: message.author.imageURL)
+        setAvatarAlignment(avatarAlignment)
         setText(message.text)
         setAttachmentPreview(for: message)
-
-        switch avatarAlignment {
-        case .left:
-            setAvatarPosition(.left)
-        case .right:
-            setAvatarPosition(.right)
-        default:
-            break
-        }
     }
-}
 
-private extension _QuotedChatMessageView {
     /// Sets the avatar image from a url or sets the placeholder image if the url is `nil`.
     /// - Parameter imageUrl: The url of the image.
-    func setAvatar(imageUrl: URL?) {
+    open func setAvatar(imageUrl: URL?) {
         let placeholder = appearance.images.userAvatarPlaceholder1
         authorAvatarView.imageView.loadImage(from: imageUrl, placeholder: placeholder)
     }
 
     /// Sets the text of the `textView`.
     /// - Parameter text: The content of the text view.
-    func setText(_ text: String) {
+    open func setText(_ text: String) {
```

Comment:
This is nowhere to be found when I checkout this branch 😅 My comment was mainly that this should be handled with `content: Content?` property instead of setting text, but I see you already did this...

Reply:
Yes, this is old code, I removed the `setText` in some commits after, probably a bad rebase 😄